app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -441,14 +450,21 @@ private class ReleaseMetricController(
     override fun start() {
         if (!isTelemetryEnabled.invoke() || initialized) { return }
 
-        services.forEach { it.start() }
+        managedScope = CoroutineScope(Dispatchers.IO)
```

Comment:
nit: as per the last PR, I'd prefer to store this in a temporary reference to avoid `!!`